ydb_sqlalchemy/sqlalchemy/__init__.py

```diff
@@ -446,6 +452,34 @@ def visit_upsert(self, insert_stmt, visited_bindparam=None, **kw):
 
 
 class YqlDDLCompiler(DDLCompiler):
+    def visit_create_index(self, create, include_schema=False, include_table_schema=True, **kw):
+        index: sa.Index = create.element
+        ydb_opts = index.dialect_options.get("ydb", {})
+
+        self._verify_index_table(index)
+
+        if index.name is None:
+            raise CompileError("ADD INDEX requires that the index have a name")
+
+        table_name = self.preparer.format_table(index.table)
+        index_name = self._prepared_index_name(index)
+
+        text = f"ALTER TABLE {table_name} ADD INDEX {index_name} GLOBAL"
+
+        text += " SYNC " if not ydb_opts.get("async", False) else " ASYNC"
```

Comment:
SYNC wrapped with space from both side, ASYNC has no space at and - it seems as typo.

Reply:
Yes, there is a redundant space after the "SYNC" word. Test doesn't fail because redundant spaces is not a error, but let's remove it